plugin-api/src/main/java/org/sonar/api/ce/measure/test/TestIssue.java

```diff
@@ -34,13 +35,14 @@
 @Immutable
 public class TestIssue implements Issue {
```

Comment:
Not related to your changes, but this class should not be an API, it's a test utility. We recently introduced a `test-fixtures` module in the repo where we gather test utilities that are useful for users. I think this class should also move there, is this something you can do while you are on it ?

Reply:
I am not sure if this is a good idea. I would prefer to keep it separately. Even small change raises a lot of comments and we need to progress here on Clean Code Taxonomy. I think the way to go would be to create a ticket in Jira for this improvement and maybe someone will work on it.

Reply:
Yes I agree. [PLUGINAPI-44](https://sonarsource.atlassian.net/browse/PLUGINAPI-44)